create-entry.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+JEKYLL_ASSETS_DIR=$HOME'/Documents/portfolio/assets/'
+JEKYLL_POSTS_DIR=$HOME'/Documents/portfolio/_posts/'
+
+FOLDER_NAME='';
+TITLE='';
+DETAIL='';
+DESCRIPTION='';
+VID=false;
```

Comment:
IMG-VID unnecessary flags if I could identify files extension over the directory I am copying

Reply:
hm? not sure what you mean by that comment.